### ISSUE TITLE
Clarify if SpanProcessor.Shutdown

### ISSUE THREAD

**Issue by jcchavezs** [open-telemetry/opentelemetry-go#1694]
At the moment Shutdown suggests it will `ForceFlush` but it isn't clear:

```go
// Shutdown is called when the SDK shuts down. Any cleanup or release of
// resources held by the processor should be done in this call.
//
// Calls to OnStart, OnEnd, or ForceFlush after this has been called
// should be ignored.
//
// All timeouts and cancellations contained in ctx must be honored, this
// should not block indefinitely.
```

I think it is more clear if this is clarified in the description, either if it is called or not.

Source https://github.com/open-telemetry/opentelemetry-go/blob/77aa218d4d8fa2ebda14a7a0b58db22fd20398b7/sdk/trace/span_processor.go#L26

**Comment by MrAlias** [comment]
If `Shutdown` calls `ForceFlush` or performs the same behavior as `ForceFlush` would seem to be up to the implementation of the interface, not something guaranteed by the interface. For instance, the `SimpleSpanProcessor` has no need to call `ForceFlush` here because it has nothing to flush.

**Comment by jcchavezs** [comment]
Let me rephrase: should we make it explicit that a flush happens if buffered data on shutdown?

**Comment by MrAlias** [comment]
I think it is fine to make it explicit that a flush happens if data is buffered, but that would be documented on the implementation not the interface is my point. The interface seems to do a good job outlining what implementations should do, but it is up to them to determine exactly how this is done and how that behavior is documented.

**Comment by zhyChesterCheung** [comment]
So is this issue finished? I saw this seems to expire and we could close it. thx~
